diff -dur zynaddsubfx-3.0.3.orig/mruby-zest-build/build_config.rb zynaddsubfx-3.0.3/mruby-zest-build/build_config.rb
--- zynaddsubfx-3.0.3.orig/mruby-zest-build/build_config.rb	2018-08-06 09:01:21.000000000 +0200
+++ zynaddsubfx-3.0.3/mruby-zest-build/build_config.rb	2018-08-07 14:32:40.000000000 +0200
@@ -92,7 +92,6 @@
   conf.cc do |cc|
       cc.include_paths << "#{`pwd`.strip}/../deps/nanovg/src"
       cc.include_paths << "#{`pwd`.strip}/../deps/pugl/"
-      cc.include_paths << "#{`pwd`.strip}/../deps/libuv-v1.9.1/include/"
       cc.include_paths << "/usr/share/mingw-w64/include/" if windows
       cc.include_paths << "/usr/x86_64-w64-mingw32/include/" if windows
       cc.flags << "-DLDBL_EPSILON=1e-6" if windows
@@ -110,7 +109,7 @@
       #linker.library_paths  << "#{`pwd`.strip}/../deps/rtosc/build/"
       linker.library_paths  << "#{`pwd`.strip}/../src/osc-bridge/"
       linker.libraries << 'osc-bridge'
-      linker.flags_after_libraries  << "#{`pwd`.strip}/../deps/libuv.a"
+      linker.flags_after_libraries  << "-luv"
       linker.flags_after_libraries  << "#{`pwd`.strip}/../deps/pugl/build/libpugl-0.a"
       linker.flags_after_libraries  << "#{`pwd`.strip}/../deps/libnanovg.a"
       if(!windows)
diff -dur zynaddsubfx-3.0.3.orig/mruby-zest-build/Makefile zynaddsubfx-3.0.3/mruby-zest-build/Makefile
--- zynaddsubfx-3.0.3.orig/mruby-zest-build/Makefile	2018-08-06 09:01:21.000000000 +0200
+++ zynaddsubfx-3.0.3/mruby-zest-build/Makefile	2018-08-07 14:31:35.000000000 +0200
@@ -10,12 +10,12 @@
 #	cd deps/pugl         && ./waf configure --no-cairo --static
 	cd deps/pugl         && ./waf configure --no-cairo --static --debug
 	cd deps/pugl         && ./waf
-	cd src/osc-bridge    && CFLAGS="-I ../../deps/libuv-v1.9.1/include " make lib
+	cd src/osc-bridge    && make lib
 	cd mruby             && MRUBY_CONFIG=../build_config.rb rake
 	$(CC) -shared -o libzest.so `find mruby/build/host -type f | grep -e "\.o$$" | grep -v bin` ./deps/libnanovg.a \
 		./deps/libnanovg.a \
 		src/osc-bridge/libosc-bridge.a \
-		./deps/libuv-v1.9.1/.libs/libuv.a  -lm -lX11 -lGL -lpthread
+		-luv -lm -lX11 -lGL -lpthread
 	$(CC) test-libversion.c deps/pugl/build/libpugl-0.a -ldl -o zest -lX11 -lGL -lpthread -I deps/pugl -std=gnu99
 
 osx:
@@ -49,20 +49,13 @@
 
 
 builddep:
-	cd deps/$(UV_DIR)    && ./autogen.sh
-	cd deps/$(UV_DIR)    && CFLAGS=-fPIC ./configure
-	cd deps/$(UV_DIR)    && CFLAGS=-fPIC make
-	cp deps/$(UV_DIR)/.libs/libuv.a deps/
 
 builddepwin:
 	cd deps/$(UV_DIR)   && ./autogen.sh
 	cd deps/$(UV_DIR)   && CFLAGS="-mstackrealign" ./configure  --host=x86_64-w64-mingw32
 	cd deps/$(UV_DIR)   && LD=x86_64-w64-mingw32-gcc make
-	cp deps/$(UV_DIR)/.libs/libuv.a deps/
 
 setup:
-	cd deps              && wget -4 $(UV_URL)
-	cd deps              && tar xvf $(UV_FILE)
 
 setupwin:
 	cd deps              && wget -4 $(UV_URL)
